lib/constants.js

```diff
@@ -32,7 +32,7 @@ const NXMasterOwnerParamType = {
 };
 
 const Role = {
-  NonMember: 0,
+  UnAssigned: 0,
```

Comment:
`Unassigned`

Reply:
`UnAssigned` is in the IMemberRoles, that's why it's being used here too